openreview/conference/webfield.py

```diff
@@ -400,3 +400,11 @@ def set_program_chair_page(self, conference, group):
             group.web = content
             return self.client.post_group(group)
 
+    def edit_global_value(self, group, global_name, old_value, new_value):
+        group.web = group.web.replace("var "+global_name+" = "+old_value+";", "var "+global_name+" = "+new_value+";")
+        return self.client.post_group(group)
+
+    def edit_global_string_value(self, group, global_name, old_value, new_value):
```

Comment:
It seems edit_global_value and edit_global_string_value are doing the same thing. Is there a difference that I am missing @pmandler ?

Reply:
Also, with a regex search, I think, we can make it work without requiring the old value.